README.md

```diff
@@ -217,6 +252,46 @@ Each cluster that will be connected must have Submariner installed within it. Yo
    |\<SERVICE_CIDR>|Service CIDR for Cluster|""|`10.43.0.0/16`|
    |\<NAT_ENABLED>|If in a cloud provider that uses 1:1 NAT between instances (for example, AWS VPC), you should set this to `true` so that Submariner is aware of the 1:1 NAT condition.|"false"|`false`|
 
+
```

Comment:
I'd move the operator first, before helm charts.

Reply:
Done